packages/expo-ui/components/Button/index.tsx

```diff
@@ -98,6 +107,9 @@ export type ButtonProps = {
   disabled?: boolean;
 };
 
+/**
+ * @hidden
+ */
```

Comment:
Are the `@hidden` annotations necessary when we aren't exporting the types from `index.ts`?

Reply:
hmm, the `types` file is the entrypoint for our `gdad` command, so it was showing up in generated types.